src/shared/components/ncTable/sections/CustomTable.vue

```diff
@@ -464,7 +471,6 @@ export default {
 
 	thead {
 		position: sticky;
-		top: 118px;
```

Comment:
Adjusted in the other PR, let's remove this from here

Reply:
already removed 🤔

Reply:
Sorry was referring remove the change of removing this line from the commit. It actually causes a conflict now which just would need to be resolved.